lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -0,0 +1,311 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class PayuLatamGateway < Gateway
+      self.test_url = 'https://stg.api.payulatam.com/payments-api/4.0/service.cgi'
+      self.live_url = 'https://api.payulatam.com/payments-api/4.0/service.cgi'
+
+      INFO_TEST_URL = 'https://stg.api.payulatam.com/reports-api/4.0/service.cgi'
+      INFO_LIVE_URL = 'https://api.payulatam.com/reports-api/4.0/service.cgi'
+
+      self.supported_countries = ['CO']
+      self.default_currency = 'COP'
+      self.supported_cardtypes = [:visa, :master, :american_express, :diners_club]
+
+      self.homepage_url = 'http://payu.com.co/'
+      self.display_name = 'PayU Colombia'      
+
+      STANDARD_ERROR_CODE_MAPPING = {}
+
+      BRAND_MAP = {
+        visa: "VISA",
+        master: "MAST",
+        american_express: "AMEX",
+        diners_club: "DINR"
+      }
+
+      def initialize(options={})
+        requires!(options, :key, :login, :account_id, :merchant_id)
+        super
+      end
+
+      # options = {
+      #   user: {identification: "123", full_name: "APPROVED", email: "test@test.com"}, 
+      #   billing_address: {street1: "123", street2: "", city: "Barranquilla", state: "Atlantico", country: "CO", zip: "080020", phone: "1234"}
+      # }
+      def purchase(money, payment, options={})
+        post = {
+          language: "es",
+          command: "SUBMIT_TRANSACTION",
+          merchant: {},
+          transaction: {},
+          test: test?
+        }
+
+        add_merchant(post)
+        add_invoice(post, money, options)
+        add_payment(post, payment)
+        add_customer_data(post, options)
+        add_addresses(post, options)        
+        add_other_transaction_data(post, options)
+        add_authorization_capture(post)
+
+        commit("purchase", post)        
+      end
+
+      # options = {order_id: "order_id", reason: "reson for the refund", transaction_id: "id of the transaction"}
+      def refund(options={})
+        post = {
+          language: "es",
+          command: "SUBMIT_TRANSACTION",
+          merchant: {},
+          transaction: {},
+          test: test?
+        }
+        add_merchant(post)
+
+        options.merge!({type: "REFUND"})
+
+        add_transaction(post, options)
+        
+        commit("refund", post)
+      end
+
+      # options = {order_id: "order_id", reason: "reson for the cancelation", transaction_id: "id of the transaction"}
+      def void(options={})
```

Comment:
`void` in ActiveMerchant is intended to reverse an `authorize`, so this doesn't seem useful without `authorize` support.